fs-storage/tests/SyncStatus.java

```diff
@@ -0,0 +1,11 @@
+/// Represents the synchronization status of the storage.
+public enum SyncStatus {
```

Comment:
We should a note in the rust code of `SyncStatus` that any change should be applied here as well